spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -109,6 +109,13 @@ sealed class SpdxExpression {
      */
     abstract fun spdxLicenses(): EnumSet<SpdxLicense>
 
+    /**
+     * Try to normalize all license IDs using a mapping containing common misspellings of license IDs. This function
```

Comment:
What does "Try" mean here? What exactly is returned if normalization fails? The unmodified `SpdxExpression`?

Reply:
Reworded and removed "try".